doc/project-plan/project-plan.rst

```diff
@@ -107,6 +107,34 @@ Milestones
      - March 31, 2021
    * - Milestone 3
      -
+         - Install Wizard improvements
+         - Increase testing coverage
+     - May 7, 2021
+   * - Milestone 4
```

Comment:
So, I don't think we need to keep adding this to the project plan. I think we can probably just put something up the top that says this was for the first 3 months (3 milestones) and use the roadmap below to track the rest. Otherwise all we're doing is keeping track of this in about 40 places. Also, each milestone should have a definition when we create them...

Reply:
good point. Will adjust.